src/views/app/folder-panel/conversations/conversation-list.tsx

```diff
@@ -51,18 +51,17 @@ const ConversationList: FC = () => {
 	const folder = useFolder(folderId);
 	const hasMore = useMemo(() => status === 'hasMore', [status]);
 
-	const loadMore = useCallback(
-		(date) => {
-			if (hasMore && !isLoading) {
-				setIsLoading(true);
-				const dateOrNull = date ? new Date(date) : null;
-				dispatch(search({ folderId, before: dateOrNull, limit: 50 })).then(() => {
-					setIsLoading(false);
-				});
-			}
-		},
-		[hasMore, isLoading, dispatch, folderId]
-	);
+	const loadMore = useCallback(() => {
+		if (hasMore && !isLoading) {
+			const date =
+				conversations?.[conversations.length - 1]?.date ?? new Date().setHours(0, 0, 0, 0);
+			setIsLoading(true);
+			const dateOrNull = date ? new Date(date) : null;
+			dispatch(search({ folderId, before: dateOrNull, limit: 50 })).then(() => {
```

Comment:
Same as above. I know that those values were already there but IMO this can be the opportunity to tidy up the code a little, and to avoid future breaks if we were in need to change the limits/range and we forget to replace one value